packages/compliance/README.md

```diff
@@ -0,0 +1,67 @@
+# Javascript client for the compliance API
+If you want to use [RedHatInsights/compliance](https://github.com/RedHatInsights/compliance) you shouldn't use get requests directly, but rather use this client to integrate with this service.
+
+## Install
+NPM
+```bash
+npm install --save @redhat-cloud-services/compliance-client
+```
+
+Or Yarn
+```bash
+yarn add @redhat-cloud-services/compliance-client
+```
+
+### Usage
+This client is using typescript and axios. Types are distributed with this package, so no need to define or install them separately.
+
+To correctly bootstrap this API you should use this config (no need to define it multiple times, just one config and reimport it anywhere you want to use it).
+```JS
+// api.js
+import axios from 'axios';
```

Comment:
The new generator works differently. For optimal use, you should use the common API factory and include only the API methods you need. There are 2 ways to initialize the factory:

```js
import { APIFactory } from '@redhat-cloud-services/javascript-clients-shared/utils'
import updatePolicy from '@redhat-cloud-services/compliance-client/dist/UpdatePolicy'

// using basepath (API is on a different origin than the browser session)
const APIInstance = APIFactory('https://foo/bar', {
  updatePolicy
}, {
  axios: {
    // axios options  
  }
})

// same origin
const APIInstance = APIFactory({
  updatePolicy
}, {
  axios: {
    // axios options  
  }
})

// using the factory
APIInstance..updatePolicy({
  id: '123',
  policyUpdate: {
    business_objective: 'new business objective',
    compliance_threshold: 80,
    description: 'new description',
  }
})

```

This will be preferred way forwards.